crates/sovereign-sdk/module-system/sov-modules-rollup-blueprint/src/lib.rs

```diff
@@ -183,14 +177,26 @@ pub trait RollupBlueprint: Sized + Send + Sync {
             },
         };
 
+        // TODO: Decide what to do with this.
```

Comment:
Can you please clarify what's wrong with the following line? The comment is a bit vague as to why we need to do something at all.

Reply:
Well. I need a storage to create: `DbProvider::new(storage.clone())`. The question where to build it - in sequencer.rs or here in in sov rollup blueprint. If inside sequencer then it already works. If inside sov rollup blueprint (which I would prefer) - we diverge from sov code more and more.

Reply:
Understood, i suggest adding what you wrote to the code comment so that it's more clear to whoever is reading. Unless you intend to resolve this TODO in this PR?

Reply:
I think yes, we need to resolve it here